### ISSUE TITLE
Missing .deb file for linux

### ISSUE THREAD

**Issue by danis101** [SnosMe/awakened-poe-trade#1365]
Hi! Since Im trying to move on linux (having best performance in poe at Kubuntu) I cannot run awakened poe Trade from AppImage without issues. I'd rather to install it via .deb than fight for like 4 days straight with  " --no-sandbox " thing, it is really annoying to run app from terminal all the time especially during gaming. Is there any way to get deb?

**Comment by sinder38** [comment]
If you don't like running from terminal you can create a .desktop shortcut manually and use a shell script to launch the app without sandbox.
Or you can probably use appImageLauncher (never used it myself)

**Comment by Ekital** [comment]
@sinder38 Manually creating a .desktop shortcut will break with every Awakened PoE Update as the name of the appimage will change automatically.

I suggest using [gearlever](https://github.com/mijorus/gearlever) for integrating AppImages as it will automatically update the .desktop file. 

Simply install gearlevel, run it, drop in your AppImage and let it make the desktop file for you automatically and done.

appimageLauncher works as well it just hasn't been maintained for a long time.

**Comment by sinder38** [comment]
@Ekital Fair point. I forgot it does this. But it is still possible to do without installing anything. You can place the AppImage into its own folder, write a bash script that executes the first AppImage file in said folder, and reference your bash script in the .desktop file.

**Here is how that bash script might look like:**
```bash
#!/bin/bash

# Navigate to the directory containing the AppImage files
cd /path/to/your/folder

# Find the first AppImage file and store it in a variable
appimage_file=$(ls *.AppImage | head -n 1)

# Check if an AppImage file was found
if [ -n "$appimage_file" ]; then
  # Make the AppImage file executable
  chmod +x "$appimage_file"

  # Run the AppImage file
  ./"$appimage_file"
else
  echo "No AppImage file found in the directory."
fi
```

**Comment by Ekital** [comment]
@sinder38 Yes, but now you have to separate each of your AppImages into a separate folder. It's a lot of work if you have a lot of them. You can probably also add some type of keyword search using regex or some sort of fuzzy finder to find the right appimage and update the desktop file.. but that's just a lot of work that can be automated without you messing around and updating your scripts lol.

Also based on the knowledge of the poster a complicated solution is not what he is looking for xD.

**Comment by danis101** [comment]
I Just solved it with gearleaver.
I dont neeed overcomplicating life with writing scripts anyway thank you guys for the input.
Deb still will be better becouse I See its way faster to run integrated apps instead appimages, I dont really get it why we cant get *.deb as lychee slicer does - they post deb and appimage and everyone is happy :)
So to everyone whos struggling gearleaver with  - - no-sandbox option is working fine.
